### ISSUE TITLE
JAWS does not read read-only text-area content when reading keys are pressed

### ISSUE THREAD

**Issue by dingbx** [FreedomScientific/VFO-standards-support#567]
### Summary

1. Go to https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_textarea_readonly
2. Tab back to the text-area on the right
3. JAWS announce the first line of the text-area and "use your reading key to read the text"
4. Press Insert + Down Arrow

### Expected result
Read the next line of the text.

### Actual result
Keep reading the first line. Also tried other reading keys other keys move the focus to another element.

#### JAWS version and build number
2021/2103/174 ILM

#### Operating System and version
Windows 10 Enterprise 
21H1

#### Browser and version:
Chrome
Version 92.0.4515.131 (Official Build) (32-bit)

**Comment by JAWS-test** [comment]
I can confirm the problem.

I can add the following further test results:

* The right and left arrow keys cannot be used to read letter by letter, there is no output at all.
* With the arrow keys up and down can not be read line by line. Instead, the virtual cursor is placed at the beginning or end of the entire HTML document.

**Comment by StephenBeckettTPGi** [comment]
Can confirm this behaviour in Chrome (FireFox seems fine).
